### PR TITLE
Update Director

### PR DESCRIPTION
**Pull request by CannonLock**
- Move to cards over table view
- Generalize the card list component
- Implement the DirectorCard

### CODE REVIEW

**Comment by CannonLock** [comment]
Screenshot 2024-04-08 at 10 30 43 AM (2)
Screenshot 2024-04-08 at 10 30 39 AM (2)

**Comment by CannonLock** [comment]
@haoming29 Notably the `No Results` helpertext leaks into the registry page where it is slightly less elegant then the previous solution. Since that is only aesthetic I will move making that look better to next release.

**Review by haoming29 — APPROVED** [review]
LGTM! One minor thought that's non-blocking

File: web_ui/frontend/components/Namespace/DirectorCard.tsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+                                                    } else {
+                                                        mutate()
+                                                    }
+
+                                                    setDisabled(false)
+                                                }}
+                                            />
+                                        }
+                                        label={!filtered ? "Active" : "Disabled"}
+                                    />
```
[the remark this answers is not in the document]
Comment (CannonLock) [reply web_ui/frontend/components/Namespace/DirectorCard.tsx:100]: https://github.com/PelicanPlatform/pelican/issues/1072

**Pull request merged**